src/main/kotlin/no/nav/syfo/aktivitetskrav/api/AktivitetskravApi.kt

```diff
@@ -74,6 +78,29 @@ fun Route.registerAktivitetskravApi(
 
             call.respond(HttpStatusCode.OK)
         }
+        post("/{$aktivitetskravParam}$forhandsvarselPath") {
+            val personIdent = call.personIdent()
+            val callId = call.getCallId()
+            val aktivitetskravUUID = UUID.fromString(call.parameters[aktivitetskravParam])
+
+            // Trenger kanskje ikke veilederIdent her?
+            // Vi må sette opp en reminder på at veileder skal få en oppgave når fristen på å svare til innbygger har gått ut.
+            // val veilederIdent = call.getNAVIdent()
+            val requestDTO = call.receive<ForhandsvarselDTO>()
+            if (requestDTO.document.isEmpty()) {
+                throw IllegalArgumentException("Forhandsvarsel can't have an empty document")
+            }
+
+            val forhandsvarsel = aktivitetskravService.sendForhandsvarsel(
+                personIdent,
+                call.getNAVIdent(),
+                aktivitetskravUUID,
+                requestDTO,
+                callId
+            )
+
+            call.respond(HttpStatusCode.Created, forhandsvarsel)
```

Comment:
Burde vi lage et `AktivitetskravVarselDTO`-objekt og returnere i responsen her i stedet for domeneobjekt? (hvis vi trenger å returnere noe)

Reply:
Hvis `AktivitetskravVarselDTO` hadde hatt noen andre properties enn domeneobjektet så tenker jeg dette hadde gitt mening, men hvis de ser helt like ut så vet jeg ikke helt hva fordelene ville vært. 
Vi ivaretar fortsatt lagdelingen ettersom domene laget er fullt synlig for api laget 😊
Enig i at med en gang de ser litt forskjellige ut så burde vi ha et dto objekt, men før dette tror jeg det går fint med domene objektet. Hva tenker du? 😊

Reply:
Greit det :+1: En egen DTO er vel avhengig av hva frontend evt trenger så vi kan godt la det være slik nå.

Reply:
Ja, tenkte også på at det kanskje hadde vært "penere" når vi først returnerer noe 🤔